gdx/src/com/badlogic/gdx/math/Intersector.java

```diff
@@ -36,6 +36,10 @@
 	private final static Vector3 v2 = new Vector3();
 	private final static FloatArray floatArray = new FloatArray();
 	private final static FloatArray floatArray2 = new FloatArray();
+	
+	private Intersector() {
```

Comment:
You can run the libgdx formatter to fix up spaces vs tabs, etc automatically (I can tell you haven't because there's no space before `()`). However, running it manually on each file is not a good use of time. We have an issue to improve this: #5698

Reply:
Yes i also recommend this command, it saves a lot of time.